jakartaee-api/pom.xml

```diff
@@ -84,27 +85,31 @@
             <groupId>jakarta.platform</groupId>
             <artifactId>jakarta.jakartaee-web-api</artifactId>
             <version>${project.version}</version>
-            <optional>false</optional>
+        </dependency>
+        <!-- Core Profile API -->
+        <!-- We specify it here directly, so it's "closer" to the user app.
+             This way it will win in Maven's resolution mechanism -->
+        <dependency>
```

Comment:
If the intention this to win in the resolution mechanism, then it should be defined before the Web Profile dependency.

Reply:
Yes, the comment is a bit misleading. The intent is to be "neutral" aka "carry through" the user's intent. If the user wants "MP's version of Jakarta Core Profile" to win, they can put the MP dependency first. If they want Jakarta's Core to win, they put Jakarta first.

This is to be consistent and intuitive with how maven resolution works, and able to specify exactly what the user intended

Reply:
I will try to find a better language for the comment

Reply:
Ok, I found better language for the comment and committed it. Thank you!

Reply:
Or just move it up one dependency?

1. Core Profile dependency
2. Web Profile dependency
3. all the rest beloging to Platform

Reply:
In an ideal world, where we could assume that there is only one single version for a spec defined as a depedency in the Profiles and the component specs themself, we would not need to reference the Core Profile here again.

In the situation, that the Web Profile makes ajustments to Core Profile dependencies (like having optional parts becoming required or build time dependencies being not part of the Core Profile set, as mentioned in #140), we need to reference this in the order of having Web Profile picked first, then having Core Profile as a second regarding the versions picked and need to verify that the additional configuration is picked (and kept) from the Web Profile too (and is not overwritten).

The comment sould point to that fact, so we know when we could remove the Core Profile dependency in a future release (where the issues mentioned above fixed) or to prevent somebody to change the order. 
@lprimak do you have a suggestion for the comment? Otherwise I can give a try.